account_statement_completion/models/completion_rules.py

```diff
@@ -28,6 +29,29 @@ class StatementCompletionRule(models.Model):
     _name = "account.statement.completion.rule"
     _description = 'Account Statement Completion Rule'
 
+    DEFAULT_VAL = """
+                    # Available variables:
+                    #-------------------------------
+                    # stmts_vals: Values of the statements as a list of dict
+                    # stmt_line: Values of the statement line as a dict
+                    # line_amount: stmt_line amount
+                    # ref: stmt_line reference
+                    # env: environment
+                    # AccMove: Odoo model "account.move"
+                    # AccMoveLine: Odoo model "account.move.line"
+                    # Partner: Odoo model "res.partner".
+                    # Payment: Odoo model "account.payment"
+                    # PaymentLine: Odoo model "account.payment.line"
+
+                    # Available compute variables:
+                    #-------------------------------
+                    # result: True if the reconcilation rule found any relevant data
```

Comment:
```suggestion
                    # result: True if the reconcilation rule found any relevant data.
                    #             This will prevent any subsequent rule to be executed on the same line.
```